website/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to Dashboards
+
+Contributions are very welcome! Just fork the repository, develop in a branch and submit a pull request.
+
+## Commit Messages
+
+We follow [conventional commits](https://www.conventionalcommits.org) when writing commit messages.
+The messages themselves should help future developers understand **why** changes were made.
+
+## Code Style
+
+We value clean code, here are some guidelines on what we consider clean code:
+
+-   Use expressive names (variables, classes, function), avoid abbreviations unless they are really common sense
+-   Small functions and classes
+-   Reasonable test coverage: We don't require 100% test coverage, but we do require tests for all non-trivial code.
+    Every piece of the code should at least be tested in the happy path.
+-   Develop code with testability in mind.
+
+### Frontend
+
+We make use of two frameworks: Astro, React, and Tailwindcss.
```

Comment:
It reads a bit funny, because it says "two", but then "Astro, React, and Tailwindcss" are 3. ;)
* Do we want to add DaisyUI to the list? Or do we consider it part of Tailwind for this purpose?
* I don't know whether framework is the right word here, but listing those is good, because it's the main ingedients to the website.

Reply:
I added daisyui to the list. It is one of our major libarries.